joomla/joomla-cms
# [4.1] Add text COM_WORKFLOW_STAGE_FORM_NEW to com_workflow.ini

Pull Request for Issue #37390 part 1 .

### Summary of Changes

Added the missing language string as discussed in the issue and removed the old one since it is not used anywhere directly or by string concatenation of the key.

### Testing Instructions

Edit a workflow stage.

In the edit view, use the action "Save & New".

After that the aria-label of the form should have changed from "Edit Stage" to "Add Stage".

### Actual result BEFORE applying this Pull Request

Aria label "COM_WORKFLOW_STAGE_FORM_ADD".

### Expected result AFTER applying this Pull Request

Aria label "Add Stage".

### Documentation Changes Required

None.
